OpenRA.Game/Manifest.cs

```diff
@@ -83,17 +83,20 @@ public readonly string[]
 		public readonly string[] SpriteFormats = Array.Empty<string>();
 		public readonly string[] PackageFormats = Array.Empty<string>();
 		public readonly string[] VideoFormats = Array.Empty<string>();
-		public readonly bool AllowUnusedTranslationsInExternalPackages = true;
 		public readonly int FontSheetSize = 512;
 		public readonly int CursorSheetSize = 512;
 
+		// TODO: This should be controlled by a user-selected translation bundle!
+		public readonly string FluentCulture = "en";
```

Comment:
No, it should be controlled by a user settings like it already is.

Reply:
it should be controlled by the bundles. The user should be able to select the language in from installed languages eventually yeah. It should give the pop-up window to reload the game same way changing display size does